src/Hazelcast.Net/Core/TaskCoreExtensions.cs

```diff
@@ -238,7 +238,7 @@ private static async Task AwaitWithTimeout(Task task, int timeoutMilliseconds, C
         {
             if (timeoutMilliseconds < 0)
             {
-                await task;
+                await task.ConfigureAwait(false);
```

Comment:
Is `CfAwait` not available here?  
+line 280.

Reply:
yes, looking into it

Reply:
this file defines `CfAwait` so as a general rule I don't use it in the file itself